cips/cip-peppermint.md

```diff
@@ -0,0 +1,36 @@
+| cip | TBD |
+| - | - |
+| title | Peppermint Network Upgrade |
+| description | Reference specifications included in the Peppermint Network Upgrade |
+| author | [@jcstein](https://github.com/jcstein) |
+| discussions-to | <https://forum.celestia.org/t/cip-v3-peppermint-network-upgrade/1826> |
+| status | Draft |
+| type | Meta |
+| created | 2024-10-01 |
+| requires | CIP-21, CIP-24, CIP-XX |
+
+## Abstract
+
+This Meta CIP lists the CIPs included in the Peppermint network upgrade.
+
+## Specification
+
+### Included CIPs
+
+- [CIP-21](./cip-21.md): Introduce blob type with verified signer
+- [CIP-24](./cip-24.md): Versioned Gas Scheduler Variables
+- CIP-XX: limiting compute per block
+
+All of the above CIPs are state breaking, and thus require a breaking network upgrade. The activation of this network upgrade will be different from future network upgrades, as described in [CIP-10](./cip-10.md).
```

Comment:
Signaling upgrades will hopefully be the same for v2 -> v3 and v3 -> v4, and onwards.
However, signaling upgrades differ from how v1 -> v2 activated (via a predetermined upgrade height).

```suggestion
All of the above CIPs are state breaking, and thus require a breaking network upgrade. The activation of this network upgrade will be different from previous network upgrades, as described in [CIP-10](./cip-10.md).
```